### PR TITLE
Better Examples (tm)

### PR DESCRIPTION
**Pull request by juanca**
## Motivation

It seems like we have been committing some examples with missing props. And the recent "all examples in 1 page" refactor implies we need better ID management.

## Acceptance Criteria

- MDS site generates without any prop-type errors
- - MDS site generates without any ID errors

## PR upkeep checklist

- [x] Change log entry
- [x] Label(s)
- [x] Assignee(s)
- [x] Deployment URL: https://mavenlink.github.io/design-system/uuid-examples/
- [x] (When ready for review) Reviewer(s)
- [x] Green Bigmaven CI check: https://github.com/mavenlink/mavenlink/pull/25047
- [ ] DevQA on Bigmaven staging (e.g. does the work need to be imported in the internal design system?)

### CODE REVIEW

File: src/components/autocompleter/autocompleter.md [hunk 1]
```diff
@@ -5,13 +5,13 @@ For a11y and additional prop info, see the `Select` component.
 ### Default examples
 ```js
 import Autocompleter from './autocompleter.jsx';
-<Autocompleter apiEndpoint='/models' id='ex-1' name='default-ex' label='Default Example' />
+<Autocompleter apiEndpoint='/models' id={uuid.v4()} name='default-ex' label='Default Example' />
```
Comment (Bloodyaugust) [inline src/components/autocompleter/autocompleter.md:8]: I love this module, and I love this approach. 👍

File: styleguide/utils/console-error.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const original = window.console.error;
+
+window.console.error = function throwOnError(...args) {
```
Comment (Bloodyaugust) [inline styleguide/utils/console-error.js:3]: Very nice. I hope this makes failures painfully obvious in the future. 😄

**Review by Bloodyaugust — APPROVED** [review]
LGTM!

File: styleguide/utils/console-error.js (hunk 2 above)
Comment (juanca) [reply styleguide/utils/console-error.js:3]: It will fail in CI! :D Yay!

**Pull request merged**